### DISCUSSION TITLE
multiple tables on one page

### DISCUSSION THREAD

**Discussion by Skao** [vincjo/datatables#96] [Q&A] [answered]
I saw a git issue related to this that @vincjo fixed and released quite some time ago. I don't understand how to manage multiple tables on one page, hence this post. What I thought would work would be to define two dataHandlers, i.e. one for each set of data per table, like this:

let userHandler = new DataHandler(users, { rowsPerPage, 5 } );
let otherHandler = new DataHandler(otherData, { rowsPerPage, 5 } );

and then use them later, for example:
<Search { userHandler } />

but Search (and everything else) expects { handler } specifically.

Is there a way to manage two tables each with its own set of data, each managed by their own DataHandler?

Thanks in advance for any assistance...

**Comment by vincjo** [comment, marked as answer]
Indeed, it may be confusing because `<Component {handler}>` is a shorhand for `<Component handler={handler}>`

In your example, passing props to components would look like this:
```svelte
<Search handler={userHandler}/>

<Search handler={otherHandler}/>
```

**Reply by Skao** [reply]
Thanks @vincjo!
